Composer/packages/lib/shared/src/constant.ts

```diff
@@ -352,7 +352,7 @@ export const QnALocales = [
 const adaptiveCardJsonBody =
```

Comment:
We should use more specialized names for all of the LG templates, these are pretty generic and can look confusing in a larger bot.

Reply:
Even if they just had something about disambiguation prepending it...like:
- OnChooseIntent_PickOne
- OnChooseIntent_GetAnswerReadBack
- etc.

Reply:
And AdaptiveCardJson should be something like ChooseIntentCard

Reply:
Thanks for the suggestion @ryanlengel , the parent template has been renamed to `ChooseIntent`, and now roughly looks like this. `designerId` guid is autogenerated to help us track certain things, but can be removed if hardcoding to a template. The `attachment` extension is to signify to users all these lg contents are part of the attachment, and helps Composer clean up resources.

```json

# ChooseIntent_cbxtWX_attachment_card(candidates)
<actual json file for card>

# ChooseIntent_cbxtWX_attachment_whichOneDidYouMean()
- I'm not sure which one you mean.
- Hmmm, I find that to be ambiguous.

# ChooseIntent_cbxtWX_attachment_pickOne()
- Can you pick one ?
- Can you help clarify by choosing one ?

# ChooseIntent_cbxtWX_attachment_getAnswerReadBack()
- See an answer from the Knowledge Base

# ChooseIntent_cbxtWX_attachment_getIntentReadBack(intent)
- SWITCH : ${intent}
- CASE: ${'QnAMatch'}
    - ${ChooseIntent_cbxtWX_attachment_getAnswerReadBack()}
- CASE : ${'GetUserProfile'}
  - Start filling in your profile(GetUserProfile intent)
- DEFAULT :
  - ${intent}

# ChooseIntent_SendActivity_s3bxdY()
- Sure, no worries.

```